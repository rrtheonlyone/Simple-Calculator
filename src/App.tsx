import React from 'react';
import './App.css';

import Calculator from './component/Calculator';

const App: React.FC = () => {
  return (
    <div className="App">
      <h2>This is a calculator</h2>
      <Calculator/>
    </div>
  );
}

export default App;
