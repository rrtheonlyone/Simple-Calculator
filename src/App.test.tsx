import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import App from './App';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);
  ReactDOM.unmountComponentAtNode(div);
});

describe('MyComponent', () => {
  it('should render correctly when rendered', () => {
    const component = shallow(<App />);
    expect(component).toMatchSnapshot();
  });
});
