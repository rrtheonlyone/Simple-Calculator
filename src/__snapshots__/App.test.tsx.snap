// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent should render correctly in "debug" mode 1`] = `
<div
  className="App"
>
  <h2>
    This is a calculator
  </h2>
  <Calculator />
</div>
`;

exports[`MyComponent should render correctly when rendered 1`] = `
<div
  className="App"
>
  <h2>
    This is a calculator
  </h2>
  <Calculator />
</div>
`;
